platform/solr/platform-solr-server-standalone/src/main/resources/solr/conf/schema.xml

```diff
@@ -383,6 +383,7 @@
             <filter class="solr.PatternReplaceFilterFactory"
                     pattern="^\p{Punct}*(.*?)\p{Punct}*$" replacement="$1"/>
             <filter class="solr.LowerCaseFilterFactory"/>
+            <filter class="solr.SynonymFilterFactory" synonyms="synonyms.txt" ignoreCase="true" expand="true"/>
```

Comment:
For clarity's sake, could we include a "synonyms.txt" in the distribution? We can map the the US word "subway" to the UK word "underground", just to show what it looks like.

Reply:
This isn't documented, but there already is a synonyms.txt file in the distro (one for each solr core actually).  We probably should document this and how to use it.  Not sure where the right place is though.

Reply:
Ask @ricklarsen. He'll know where to put it.

Reply:
Documentation added in the solr configuration section